lib/src/rules/prefer_function_declarations_over_variables.dart

```diff
@@ -53,8 +53,12 @@ class _Visitor extends SimpleAstVisitor {
 
   @override
   visitVariableDeclaration(VariableDeclaration node) {
-    if (node.initializer != null && node.initializer is FunctionExpression) {
-      rule.reportLint(node);
+    if (node?.initializer is FunctionExpression) {
```

Comment:
nit: I don't think the `node` can be `null`, so the `?.` shouldn't be needed.

Reply:
done